scripts/tests/chiptest/__init__.py

```diff
@@ -137,7 +137,7 @@ def _GetInDevelopmentTests() -> Set[str]:
         "Test_AddNewFabricFromExistingFabric.yaml",     # chip-repl does not support GetCommissionerRootCertificate and IssueNocChain command
         "TestEqualities.yaml",              # chip-repl does not support pseudo-cluster commands that return a value
         "TestExampleCluster.yaml",          # chip-repl does not load custom pseudo clusters
-        "TestClientMonitoringCluster.yaml",  # Client Monitoring Tests need a rework after the XML update
+        "TestIcdManagementCluster.yaml",  # Client Monitoring Tests need a rework after the XML update
```

Comment:
This line disable the yamls tests for certain CI runs. If the `TestIcdManagementCluster.yaml` passes, the line can simply be deleted.